Sources/TuistKit/Services/GraphService.swift

```diff
@@ -288,3 +290,99 @@ extension ProjectAutomation.Scheme {
         return ProjectAutomation.Scheme(name: scheme.name, testActionTargets: testTargets)
     }
 }
+
+extension ProjectAutomation.Settings {
```

Comment:
I'd not cargo-cult defining extensions in `GraphService.swift` because it's not apparent that a file named that way contains extensions with static constructors. We can instead:
- `ProjectAutomationModels+FromGraphModels.swift`

Reply:
extracted all to new file